arangod/Aql/AllRowsFetcher.cpp

```diff
@@ -101,6 +116,7 @@ std::pair<ExecutionState, SharedAqlItemBlockPtr> AllRowsFetcher::fetchBlock() {
 
 std::pair<ExecutionState, SharedAqlItemBlockPtr> AllRowsFetcher::fetchBlockForModificationExecutor(
     std::size_t limit = ExecutionBlock::DefaultBatchSize()) {
+  // TODO FIXME
```

Comment:
What needs to be fixed?

Reply:
This function, and I simply did NOT do it.
Thanks for spotting